src/main/java/it/gov/pagopa/pu/pagopapayments/service/paymentsreporting/PaymentsReportingService.java

```diff
@@ -0,0 +1,64 @@
+package it.gov.pagopa.pu.pagopapayments.service.paymentsreporting;
+
+import gov.telematici.pagamenti.ws.NodoChiediElencoFlussiRendicontazione;
+import gov.telematici.pagamenti.ws.NodoChiediElencoFlussiRendicontazioneRisposta;
+import it.gov.pagopa.pu.organization.dto.generated.Broker;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import it.gov.pagopa.pu.pagopapayments.connector.soap.NodeForPaClient;
+import it.gov.pagopa.pu.pagopapayments.dto.BrokerForNodoPaDTO;
+import it.gov.pagopa.pu.pagopapayments.dto.generated.ReportingIdDTO;
+import it.gov.pagopa.pu.pagopapayments.exception.ApplicationException;
+import it.gov.pagopa.pu.pagopapayments.mapper.PaymentsReportingIdMapper;
+import it.gov.pagopa.pu.pagopapayments.service.broker.BrokerService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+public class PaymentsReportingService {
+
+  private final NodeForPaClient nodeForPaClient;
+  private final BrokerService brokerService;
+
+  public PaymentsReportingService(NodeForPaClient nodeForPaClient, BrokerService brokerService) {
+    this.nodeForPaClient = nodeForPaClient;
+    this.brokerService = brokerService;
+  }
+
+  public List<ReportingIdDTO> getReportingList(Long organizationId, String accessToken) {
+
+
+    BrokerForNodoPaDTO brokerForNodoPaDTO = brokerService.getBrokerForNodoPaDTOByOrganizationId(organizationId, accessToken);
+    NodoChiediElencoFlussiRendicontazione request = getNodoChiediElencoFlussiRendicontazione(brokerForNodoPaDTO);
+
+    NodoChiediElencoFlussiRendicontazioneRisposta response = nodeForPaClient.nodoChiediElencoFlussiRendicontazione(request, brokerForNodoPaDTO.getBrokerApiKeys().getSyncKey());
+
+    if (response.getFault() != null) {
+      throw new ApplicationException("Error during the call to the payment node " + response.getFault().getFaultCode());
+    }
+
+    List<ReportingIdDTO> reportingList = new ArrayList<>();
+
+    response.getElencoFlussiRendicontazione().getIdRendicontaziones().forEach(idRendicontazione -> reportingList.add(PaymentsReportingIdMapper.map(idRendicontazione)));
```

Comment:
move SOAP request and response extract on the client class itself
let's keep the Service unaware of the protocol details